core-it-suite/src/site/apt/index.apt.vm

```diff
@@ -52,11 +52,9 @@ mvn -Preporting site
  which obviously can't resolve anything. This setup using a file-based dummy repo also helps execution time,
  because this repo produces (expected) <<<404>>>s much faster than a HTTP-based repo.
 
- The one place where access to Central is desired is in the <<<MavenITBootstrapTest>>>
- ({{{./xref-test/org/apache/maven/it/MavenITBootstrapTest.html}src}},
- {{{./testapidocs/org/apache/maven/it/MavenITBootstrapTest.html}javadoc}}),
- which doesn't really test anything but just primes the local repo with any artifacts the ITs will need:
- you can see {{{./bootstrap.html}here}} the list of plugins and artifacts that are fetched during bootstrap.
+ The one place where access to Central is desired is by adding artifacts to the
+ {{{https://github.com/apache/maven-integration-testing/blob/master/core-it-suite/src/test/resources/bootstrap.txt}bootstrap.txt}} file,
+ which is used to prime the local repo with any artifacts the ITs will need.
```

Comment:
Should this file have some order? Alphabetically, chronologically?

Reply:
The order is not important, but I'll add a comment so that people should keep it alphabetically sorted, this is easier to avoid duplication.

Reply:
Correct.